### ISSUE TITLE
New CSS styling option on Images & Banner Style color change

### ISSUE THREAD

**Issue by NCalexiaeche** [Netcentric/cn-website#285]
Acceptance Criteria:
- [x] Remove darkening filter on 
- [x] Enable colors settings on components background (ex: Social media cards)
- [x] Enable colors settings on (Cards) components for titles, subtitles and body text to white when background colours are medium or dark.
<img width="1263" alt="Screenshot 2023-02-10 at 14 38 51" src="https://user-images.githubusercontent.com/120196152/218105615-cbc81ba2-ef99-447f-8717-7168498be59e.png">
<img width="415" alt="Screenshot 2023-02-10 at 14 39 46" src="https://user-images.githubusercontent.com/120196152/218105792-1e292bee-d4ae-49cf-bb57-177f39a2e2da.png">

**Comment by NCalexiaeche** [comment]
Change text color to white. Replace the background color #6AA2DC (because it doesn't passes accesiibility) for #2E308E

**Comment by NCalexiaeche** [comment]
@jonatan-lledo-netcentric Let's make sure we cover all the different colors from the brand guidelines : https://expertise.netcentric.biz/cognizant-netcentric-brand-guidelines#home

**Comment by pedrogbot** [comment]
As we have discussed on the slack channel, by applying the same component on photos and colours it will work on colours but on photos it will look too bright and the text will not be legible, we should apply one solution for colours (which is the new brightness that is being developed) and create a new flavour for images, so they look correctly

**Comment by NCalexiaeche** [comment]
Please use those colors in the banners @sonaldhekale 
![Screenshot 2023-03-16 at 08 57 29](https://user-images.githubusercontent.com/120196152/225551041-0260c311-74d3-4c0c-9ef8-2b645d1c0776.png)
![Screenshot 2023-03-16 at 08 58 22](https://user-images.githubusercontent.com/120196152/225551227-e897a1aa-b06b-4a06-a5ef-f937014e7221.png)

**Comment by NCalexiaeche** [comment]
This ticket is only looking at:

- Grey Background Filter on Images
- Banner bright blue, purple text
Here are the target style : https://www.figma.com/file/OeE1zEoK84ckQpsAVMP6Ps/Modifications-(Copy)?node-id=1039-13365&t=SFt1G1cAZXXrIRKW-0

Another ticket will be created to change the cards from images to a specific background black that need to be created.

**Comment by NCalexiaeche** [comment]
Before: https://main--cn-website--netcentric.hlx.page/
After: https://285-accessblity-banner-text--cn-website--netcentric.hlx.page/

**Comment by DaphneKoren** [comment]
Tested the light blue Banner on:
 https://main--cn-website--netcentric.hlx.page/how-we-work/communities
https://main--cn-website--netcentric.hlx.page/what-we-do/experience-operations
https://285-new-css-styling-option-on-cards--cn-website--netcentric.hlx.page/how-we-work/virtual-squads
BG colour found is #92bbe6 √
remaining on the ticket 

- [ ] Solid background cards
- [ ] Image background cards 

<img width="1324" alt="Screenshot 2023-04-04 at 15 45 17" src="https://user-images.githubusercontent.com/129506609/229815287-ccbd510b-62ab-4a5b-8adc-15c4b9af7aaf.png">

**Comment by DaphneKoren** [comment]
The following  tickets were created for the cards 
https://github.com/Netcentric/cn-website/issues/321
https://github.com/Netcentric/cn-website/issues/323
